mars/dataframe/arithmetic/floordiv.py

```diff
@@ -75,16 +77,14 @@ def _set_inputs(self, inputs):
 
 
 def floordiv(df, other, axis='columns', level=None, fill_value=None):
-    if isinstance(other, (DATAFRAME_TYPE, SERIES_TYPE)) or np.isscalar(other):
-        op = DataFrameFloorDiv(axis=axis, level=level, fill_value=fill_value, lhs=df, rhs=other)
-    else:
-        raise NotImplementedError('Only support div with dataframe, series or scalar!')
+    if isinstance(other, (list, tuple, np.ndarray, TENSOR_TYPE)):
+        other = Series(other)
+    op = DataFrameFloorDiv(axis=axis, level=level, fill_value=fill_value, lhs=df, rhs=other)
     return op(df, other)
 
 
 def rfloordiv(df, other, axis='columns', level=None, fill_value=None):
-    if isinstance(other, (DATAFRAME_TYPE, SERIES_TYPE)) or np.isscalar(other):
-        op = DataFrameFloorDiv(axis=axis, level=level, fill_value=fill_value, lhs=other, rhs=df)
-    else:
-        raise NotImplementedError('Only support div with dataframe, series or scalar!')
+    if isinstance(other, (list, tuple, np.ndarray, TENSOR_TYPE)):
```

Comment:
The piece of code is used quite a few times, how about add a util?

Reply:
Fixed.